vamb/__main__.py

```diff
@@ -1404,14 +1422,16 @@ class TaxometerArguments(BasicArguments):
     def __init__(self, args):
         super(TaxometerArguments, self).__init__(args)
         self.taxonomy_options = TaxonomyOptions(
-            taxonomy_predictions_path=None,
             taxonomy_path=args.taxonomy,
             no_predictor=False,
+            column_contigs=args.column_contigs,
+            column_taxonomy=args.column_taxonomy,
+            delimiter_taxonomy=args.delimiter_taxonomy,
         )
         self.predictor_training_options = PredictorTrainingOptions(
             nepochs=args.pred_nepochs,
             batchsize=args.pred_batchsize,
-            batchsteps=args.pred_batchsteps,
+            batchsteps=[],
```

Comment:
Why remove this? Is this redundant w.r.t. DAdaptation, or hasn't it been verified to be better than no batch steps?

Reply:
redundant